src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,137 @@
+.page__content {
+  width: 944px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: auto;
+}
+
+.product {
```

Comment:
Rename your file to .../blocks/product.scss
Blocks name must be equals to name .scss files.